src/utils/addresses.ts

```diff
@@ -16,10 +16,54 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
+export type AddressInfo = {
+  index: number
+  isMain: boolean
+  label?: string
+  color?: string
+}
+
 export const checkAddressValidity = (address: string) => {
   const match = address.match(/^[1-9A-HJ-NP-Za-km-z]+$/)
 
   if (match === null) return false
 
   return match[0] === address && address
 }
+
+export const loadStoredAddressesInfoOfAccount = (username: string) => {
```

Comment:
Maybe we could define (cast) the expected return type for a more type strict approach?

Reply:
good one, done